m4_sessions/1706-inning/group_six.md

```diff
@@ -1,9 +1,41 @@
 # Group Six
 
-### Session Title:
+### Session Title: 
+#### Code Schools (working title: we'll create something saucier)
 
 ### Session Outcomes: 
+-Reaffirming that we're in the right place: Turing is hard and we all have disparate backgrounds, but we're on the right track for us
+-Insight into the direction of coding/bootcamps: how did we get here and where is this trend going?
+-Be able to identify efficiencies and defficiencies of coding school alums vs CS degree holders
 
 ### Session Outline:
+-give SHORT presentation
+  -topics: code school trend, benefits/downsides of code school, is code school the next blue collar economy? Is the junior dev market saturated, how are we better prepared than CS degree holders
+-break into groups of 4 or 5, each gear-up leader will be in a group
+-each group discuss the same topic for ~7 minutes at a time
+-discuss severaldifferent topics, mixing up the groups
```

Comment:
I'd suggest coming back together whole group at the end for some conversation or at least to be able to share some take aways.  You could ask everyone to share their opinion on code schools or ask everyone to share how they are feeling about the comp sci/code school trade off.